src/mono/msbuild/android/build/AndroidBuild.targets

```diff
@@ -141,19 +149,6 @@
       <AndroidLibraryMinApiLevel Condition="'$(AndroidLibraryMinApiLevel)' == ''">21</AndroidLibraryMinApiLevel>
     </PropertyGroup>
 
-    <NdkToolFinderTask
```

Comment:
Did you happen to test library mode with this change?

Reply:
+1 :) ?

Reply:
We had an offline discussion with @steveisok about this. We agree that this should be handled as a follow up: https://github.com/dotnet/runtime/issues/110757